.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           node-version: 20
           cache: npm
       - name: "Install application"
-        run: npm install --ignore-scripts
+        run: npm install --ignore-scripts --legacy-peer-deps
```

Comment:
Why is this needed? This repo (in contrast to the main repo) shouldn't have any sec bulbs in it and the dependencies should install cleanly